lib/qbo_api.rb

```diff
@@ -57,7 +57,7 @@ def connection(url: get_endpoint)
       add_authorization_middleware(faraday)
       faraday.request :url_encoded
       faraday.use FaradayMiddleware::RaiseHttpException
-      faraday.response :detailed_logger, QboApi.logger if QboApi.log
+      faraday.response :detailed_logger, QboApi.logger, "[Quickbooks]" if QboApi.log
```

Comment:
When you go through the QuickBooks App Store marketing review Intuit enforces that all references to Quickbooks be "QuickBooks" and now I am completely brainwashed. So if you could make all references to "QuickBooks" I'd sleep better.